superset/dashboards/dao.py

```diff
@@ -41,21 +42,28 @@ class DashboardDAO(BaseDAO):
 
     @classmethod
     def get_by_id_or_slug(cls, id_or_slug: Union[int, str]) -> Dashboard:
```

Comment:
should we change this method name to `get_by_id_slug_or_uuid`?

Reply:
maybe a better naming could just be `get_by_identifier` but it could touches too many files if we want to change every `id_or_slug` to `identifier. Note that uuid is also id in general though. So I think this naming should be fine.